src/SonarScanner.MSBuild.PreProcessor/PreProcessor.cs

```diff
@@ -69,6 +69,7 @@ private async Task<bool> DoExecute(ProcessedArgs localSettings)
         logger.ResumeOutput();
         InstallLoaderTargets(localSettings);
         var buildSettings = BuildSettings.GetSettingsFromEnvironment();
+        logger.LogInfo(Resources.MSG_ProjectBaseDirChange);
```

Comment:
FYI we might not always want to log this message.
There is an open conversation [here](https://docs.google.com/document/d/12ln-WiSmN6ilILm2w7a5quf-YPDl8iS0aj8B-OCPawk/edit?disco=AAABVg37b0Y), at the end of this comment chain.

Reply:
Oh I missed that, thanks, I'll convert to draft until we reach a conclusion